ERDDAP_timeseries_explorer.ipynb

```diff
@@ -15,7 +15,35 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from erddap_app.layout import layout"
```

Comment:
The header of this notebook has some outdated test. Do you mind removing it in this PR? This notebook no longer uses any tech from Jason Grout's talk.

Reply:
I removed the outdated information from the header. However, if you prefer, I can just take out the entire header.

Reply:
Let's leave that for later. We'll need a new one.